src/containers/MasonryLayout.jsx

```diff
@@ -51,6 +83,11 @@ export class MasonryLayout extends Component {
     );
   }
 
+  /**
+   * Renders the component
```

Comment:
Some of these comments are unnecessary. Eg. this one. `renders a component` doesn't help anyone